src/screens/Assets/investments/InvestmentListItem.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { BigNumber } from 'bignumber.js';
+import styled from 'styled-components/native';
+
+// Components
+import FiatValueView from 'components/modern/FiatValueView';
+import FiatChangeView from 'components/modern/FiatChangeView';
+import Image from 'components/Image';
+import Text from 'components/modern/Text';
+
+// Selectors
+import { useFiatCurrency } from 'selectors';
+
+// Utils
+import { useThemedImages } from 'utils/images';
+import { spacing } from 'utils/variables';
+
+type Props = {|
+  title: ?string,
+  iconUrl: ?string,
+  value: BigNumber,
+  change?: ?BigNumber,
```

Comment:
Included in other comments re prop and value type both being `maybe`.

Reply:
P. S. Also noticed that other `ListItem` components have it defined with single `maybe`.